click-to-reveal/script.js

```diff
@@ -0,0 +1,56 @@
+const questions = [
+    {
+        'question': 'Have you heard about the chocolate record player? ',
+        'answer': 'It sounds pretty sweet.'
+    },
+    {
+        'question': 'What do you call a factory that makes okay products?',
+        'answer': 'A satisfactory.'
+    },
+    {
+        'question': 'What did one wall say to the other?',
+        'answer': "I'll meet you at the corner."
+    }
+]
+
+window.addEventListener("DOMContentLoaded", (event) => {
+    console.log("DOM fully loaded and parsed");
+
+    // step 1 get all the buttons 
+    // step 2 loop through all the buttons and add event listeners 
+
+    const buttons = document.getElementsByTagName('button');
+    console.log(buttons);
+
+    let y = 99
```

Comment:
What does this 99 stand for? Random number likes this are dangerous and at minimum should have comments.